packages/angular/projects/ui-angular/package.json

```diff
@@ -4,19 +4,21 @@
   "scripts": {
     "build": "yarn --cwd ../../ build",
     "dev": "yarn --cwd ../../ dev",
-    "test": "yarn --cwd ../../ test"
+    "test": "yarn --cwd ../../ test",
+    "generate:css": "npx tailwindcss -i ./styles/component.tailwind.css -o ./styles/component.css"
```

Comment:
Angular libraries built with `@angular/cli` does not support tailwind: https://github.com/ng-packagr/ng-packagr/issues/1943. There are some workarounds in the thread, but the simplest solution I found was to geerate the css directly as we only use tailwind for authenticator styling. Added a command here to keep track of how I generated it. 

It would be nice if @aws-amplify/ui can output a `authenticator.css` that has tailwind preprocessed so that it's readily usable by all frameworks.